index.rst

```diff
@@ -72,30 +72,31 @@ Prerequisites
     *  **Clean** environment, only test-related images, containers
        and dependent services (besides ``docker -d``) at the start
        of **every** Autotest run.
-    *  Docker installation (with ``docker -d`` running at startup)
+    *  Docker installation (with docker daemon running)
     *  Default settings for Docker on your platform/OS,
-       unless otherwise noted.
+       including storage (LVM thin-pool + XFS).
     *  Sufficient available temporary storage for multiple copies of the
        test-related images/container content.
 
```

Comment:
Sigh, might be worth adding "run as root" as prerequisite, including an explanation of why (I can't remember the full reasons, but I do remember spending a lot of time trying to get permissions right but still not having any luck).